store.go

```diff
@@ -137,7 +130,8 @@ func (c *controller) getNetworksFromStore() ([]*network, error) {
 			ec := &endpointCnt{n: n}
 			err = store.GetObject(datastore.Key(ec.Key()...), ec)
 			if err != nil {
-				return nil, fmt.Errorf("could not find endpoint count key %s for network %s while listing: %v", datastore.Key(ec.Key()...), n.Name(), err)
```

Comment:
I don't think we should convert this to just log. If there is a network object the equivalent endpoint cnt object should be present. It's an error otherwise.